setup/installation.md

```diff
@@ -46,39 +46,37 @@ kubectl config set-context --current --namespace=coder
    `helm search repo coder -l`)
 
    ```console
-   helm install coder coder/coder --namespace coder
+   helm install coder coder/coder -n coder --version=<VERSION>
    ```
 
    **Steps 3-5 are optional for non-production deployments.**
 
-1. Get a copy of your Helm chart so that you can modify it; you'll need to
-   modify the Helm chart to update your PostgreSQL databases (step 4) and enable
-   dev URLs (step 5):
+1. Get a copy of your Helm config values so that you can modify it; you'll need
+   to modify the Helm chart to update your PostgreSQL databases (step 4) and
+   enable dev URLs (step 5):
 
-   a. Get a copy of your existing Helm chart and save it as `current-values.yaml`:
-   `helm show values coder/coder > current-values.yaml`
+   a. Get a copy of your existing Helm values and save it as `values.yaml`:
+   `helm get values coder -n coder | tail -n +2 > values.yaml`
 
-   b. Edit the `current-values.yaml` file as needed. Be sure to remove the lines
-   that you are _not_ modifying, otherwise the contents of `current-values.yaml`
-   will override those in the default chart.
+   b. Edit the `values.yaml` file as needed.
```

Comment:
Should we keep:

> Be sure to remove the lines   that you are _not_ modifying, otherwise the contents of `current-values.yaml`   will override those in the default chart.

Or does it really not have any relevance?

Reply:
No more relevance

Reply:
Was that a change on the helm side?

Reply:
We no longer get a full copy of the values file for the user, so they don't have to comment out anything